### PR TITLE
build(mkdocs): fix mathjax rendering

### PR DESCRIPTION
**Pull request by soblin**
## Description

Equations [in this page](https://autowarefoundation.github.io/autoware-documentation/main/how-to-guides/integrating-autoware/creating-vehicle-interface-package/customizing-for-differential-drive-model/) were not rendered, so I fixed it.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [X] I've confirmed the [contribution guidelines].
- [X] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by soblin** [comment]
It looks like regardless of this PR the documentation page should be reloaded to render math equations.

File: mkdocs.yaml
Comment (kenji-miyake) [inline mkdocs.yaml:1]: @soblin Could you update this pull request to be the same as https://github.com/autowarefoundation/autoware.universe/pull/3846?

https://squidfunk.github.io/mkdocs-material/reference/math/#mathjax
image
image

Comment (kenji-miyake) [reply mkdocs.yaml:1]: Or is `enable_dollar_delimiter: True` another solution?

File: mkdocs.yaml [hunk 1]
```diff
@@ -75,7 +77,8 @@ markdown_extensions:
   - plantuml_markdown:
       server: http://www.plantuml.com/plantuml
       format: svg
-  - pymdownx.arithmatex
+  - pymdownx.arithmatex:
+      generic: true
```
Comment (kenji-miyake) [inline mkdocs.yaml:81]: Could you tell me the reference documentation for this?

Comment (kenji-miyake) [reply mkdocs.yaml:81]: And what was the problem? Is this also related to the MathJax issue?

Comment (soblin) [reply mkdocs.yaml:81]: Oh I did not pay much attention to it. According to [Arithmatex documentation](https://facelessuser.github.io/pymdown-extensions/extensions/arithmatex/#generic-output-format), this option seems to offer different html rendering approach of mathjax equation. I guess `generic: true` may not be a necessary change this time.

Comment (kenji-miyake) [reply mkdocs.yaml:81]: So do you want to remove it?

```suggestion
```

Comment (soblin) [reply mkdocs.yaml:81]: I tested on my local machine, and found out `genetic: true` is a must option for rendering mathjax. Without this option they were not rendered even if I reloaded the page many times.
I think I can readily ship this PR once I checked https://autowarefoundation.github.io/autoware-documentation/pr-380/ !

Comment (kenji-miyake) [reply mkdocs.yaml:81]: Understood. Thank you for testing it!

**Review by kenji-miyake — APPROVED** [review]

**Comment by soblin** [comment]
I confirmed that compared to current main branch page, the deployed page works on
- firefox@iPhone
- google chrome@Ubuntu22
by visiting [how to guides](https://autowarefoundation.github.io/autoware-documentation/main/how-to-guides/) and then moving to "Customizing for differential drive vehicle" part manually and checking the mathjax

**Pull request merged**